extensions/interactions/SetInput/directives/oppia-response-set-input.component.ts

```diff
@@ -20,22 +20,24 @@
  * followed by the name of the arg.
  */
 
-require('services/html-escaper.service.ts');
+import { Component, Input, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { HtmlEscaperService } from 'services/html-escaper.service';
+@Component({
+  selector: 'oppia-response-set-input',
+  templateUrl: './set-input-response.component.html'
+})
+export class ResponseSetInputComponent implements OnInit {
+  @Input('answer') answerWithValue;
+  answer;
```

Comment:
Why this needs to be here?

Reply:
Displayed answer... Basically, I want to keep the view variables separate.